pyproject.toml

```
@@ -56,7 +56,7 @@ classifiers = [
 dependencies = [
     "minimega==2.9",
     "colorama<=0.4.6",
-    "grpcio>=1.49.0,<=1.68.0",
+    "grpcio>=1.49.0,<=1.69.0",
```

```suggestion
    "grpcio>=1.49.0,<=1.67.0",
```

Due to: https://github.com/grpc/grpc/issues/38282